src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -464,17 +341,17 @@ public boolean incrementBlock()
     }
 
     /**
-     * Checks if the block in the world is the same as what is in the schematic.
+     * Checks if the block in the world is the same as what is in the structure.
      *
-     * @return true if the schematic block equals the world block.
+     * @return true if the structure block equals the world block.
      */
-    public boolean doesSchematicBlockEqualWorldBlock()
+    public boolean doesStructureBlockEqualWorldBlock()
     {
-        IBlockState schematicBlockState = schematicWorld.getBlockState(this.getLocalPosition());
-        Block schematicBlock = schematicBlockState.getBlock();
+        IBlockState structureBlockState = structure.getBlockState(this.getLocalPosition());
+        Block structureBlock = structureBlockState.getBlock();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'structureBlock' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)